src/panel/poi/blocks/Images.jsx

```diff
@@ -12,10 +12,9 @@ const ImagesBlock = ({
   }
 
   return <div className="poi_panel__pictures">
-    <h5 className="poi_panel__pictures_title">
```

Comment:
`h5` was clearly not the good choice anymore, but it would be nice to keep some kind of semantic hierarchy. As the POI title is `h2`, logically this should be a `h3`. What do you think? Or we decide we address all this globally on the app later?

Reply:
I think it's better to define our typography once and for all across the app :)